client/service.go

```diff
@@ -142,13 +142,25 @@ func (svr *Service) keepControllerWorking() {
 	maxDelayTime := 20 * time.Second
 	delayTime := time.Second
 
+	cutoffTime := time.Now().Add(time.Second * time.Duration(60))
+	reconnectDelay := time.Second
+
 	for {
 		<-svr.ctl.ClosedDoneCh()
 		if atomic.LoadUint32(&svr.exit) != 0 {
 			return
 		}
 
 		for {
+			time.Sleep(reconnectDelay)
```

Comment:
Move this outside this loop. It's not designed for login fail case.

Reply:
fix